### ISSUE TITLE
Numbered List Option

### ISSUE THREAD

**Issue by Recover-Zone** [pojome/elementor#6502]
## Prerequisites

- [x] I have searched for similar features requests in both open and **closed** **(One closed ticket found #5641)** tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
On our current website as we were going through the designing process we came across an issue that seems as though it could be a simple fix. That is we desired to incorporate a numbered list to allow for our website users to clearly understand a process we were outlining, but much to our surprise found that currently there is no option on Elementor for numbered lists. In our case we settled for a bulleted list, which by no means looks poor, but could greatly increase readability if we were able to insert a numbered list. 

**Describe the solution you'd like**
Simply put we would love for Elementor to add a numbered list option to the plugin. 

**Describe alternatives you've considered**
We have thought of incorporating a text block and adding a numbered list through that, but do not want to sacrifice flexibility surrounding the list. Adding a text block would complicate our format, and would limit our design even more. 

**Additional context**
We hope this is something you will be able to add to Elementor. Thanks for your time in checking out our feature request.

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

Thanks for the feedback.

**Comment by sdvizac** [comment]
By searching online for this solution, I came across this post. Elementor team could just add numbers icon with fontawesome, I was surprised also that something simple as this have to do workaround with additional columns.  

Separate from this, on my new project, I also realized that read more, read less functionality isn't available but only on working with posts. It would be handy to add this as well. 

Best regards

**Comment by justvipera** [comment]
I would love a numbered list element as well!

**Comment by theDanielJLewis** [comment]
I, too, would like to see this. In addition to possibly supporting the Font Awesome numbers, another way could be a universal upgrade to SVG icons so that wherever an SVG icon can be used in Elementor, a number could be used and styled instead.

This could be for features, icons, and more. The options I would expect to see would be:

- Number/character (allowing any character entry, not forcing a numbered list) 
- Typography (font, size, color, hover changes)
- Shape (circle, square, rounded corners)
- Background (color, transparency, hover changes)
- Border (color, transparency, width, hover changes)
- Size (probably in simple diameter, because padding would have weird results with some numbers)

**Comment by schazzw** [comment]
I would like to see this too, was just trying to add a numbered list and surprised to find that I couldn't without custom code to make it look good.

**Comment by MauriceW2** [comment]
HUH??? Not available? Something **_SO incredibly standard_** is NOT available? It's like buying car and then discover that it has no engine.

A numbered list is a super standard thing EVERYONE uses on every site.

It's really too ******* for words that this is missing... and that the comment is that it's not supported yet... I'm really starting to doubt these Elementor people. Where the heck are their brains?

**Comment by theDanielJLewis** [comment]
I wouldn't say Elementor is brainless. I chose Elementor after trying other builders because many of Elementor's features seemed far smarter than the competitor's.

I think this is simply one of those needs that isn't obvious until people like us voice our desire for them.

We already have the ability to make numbered lists with a simple text block. So that's worked for years for many people. But what I and others are interested in is something with a much nicer design flexibility like icon lists and feature lists offer. And based on Elementor's previous implementations of customer-submitted ideas, I'm sure this one is only a matter of time before this issue will be marked resolved.

**Comment by vgstef** [comment]
Me too, I'd like this option. I often have to make list, some times numbered, sometimes with bullets. That'd be really useful if the actuel List widget would allow both.

**Comment by aydavis** [comment]
I would also like to see a numbered list widget added to the Elementor library. Alternatively, you could add numbers to the icon library. Thanks!

**Comment by silvanet** [comment]
I was trying to edit a pre-existing page that had properly numbered list items. When I opened it in Elementor only the first item in the list stayed numbered. The others all became dots. Trying to change them to numbered list items was ridiculously difficult. When I selected a dotted item and changed tit to numbered, Elementor began again at number 1. The same with all succeeding changes. I can change the  items but only to all have the number 1. Elementor does not seem to have a feature to keep numbering from the prior number on the list! How useful is that?

**Comment by vgstef** [comment]
Why is this issue closed? This should be labeled Feature request. No?

**Comment by jthouse27** [comment]
Yeah, this really needs to happen. I've been waiting for over a year. Shocked its not already an option. It would be super simple if free icon libraries included numbers, but they don't for some reason, and I don't expect that they will anytime soon. Let's fix this Elementor :)

**Comment by msciocia** [comment]
I created a Github account just to come in here and request this feature be added.  PLS!

**Comment by silvanet** [comment]
Why does this happen everywhere? Not just Elementor, but everywhere they close threads as if they were resolved. That is one of the most frustrating treatments I find in every single support forum or github repo I have ever visited for help. Anyone else run into that?

**Comment by alexey4453** [comment]
I'm definitely having hard time trying to find out how to build a numbered feature list for a WC product in Elementor. Even though I have the pro version.

**Comment by Jimmi08** [comment]
> I definitely had hard time trying to find out how to build a numbered feature list for a WC product in Elementor. Even though I have the pro version. Added: I finally found a plugin that suits for my needs. It is Unlimited Elements plugin for Elementor. After installation and activation of the plugin inside it you need to install the list widget (from its widget category Marketing Widgets). Using the list widget is free and it allows auto-numeration and columns.

Thank you so much. Without your help I wouldn't know how to solve this and how to find this exact widget.  I must say it is a bit hard if you are not familiar with this plugins.

**Comment by theDanielJLewis** [comment]
I'm surprised and disappointed that they still haven't added this after 5 years.
